CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [6.3.0] - 2022-07-11
+
```

Comment:
Why is the changelog empty?

Reply:
Crap, looks like a bad rebase from the PR. Let me manually fix it. 1 min

Reply:
Updated